src/App.js

```diff
@@ -10,10 +11,35 @@ const tabs = [
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedTab: tabs[0],
+  }
+
+  onTabSelected = (tabId) => {
+    const newTab = tabs.find(tab => tab.id === tabId);
+
+    this.setState({
+      selectedTab: newTab,
+    });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <h1>
+          Selected tab is&nbsp;
+          {this.state.selectedTab.title}
+        </h1>
+
+        <Tabs
+          tabs={tabs}
+          selectedTab={this.state.selectedTab}
```

Comment:
```suggestion
          selectedTabId={this.state.selectedTab.id}
```

> Pass the `selectedTabId` as a prop to the Tabs, the specified tab should be selected if possible (otherwise the first tab is selected)

You are passing the whole `tabs` array. You may pass only `id` and find needed tab there by id